ecommerce/forms.py

```diff
@@ -13,3 +13,31 @@ class Meta:
         model = Profile
         fields = ('birthday','address')
         labels = {'birthday':_('birthday'),'address':_('address')}
+
+class ReviewForm(forms.ModelForm):
+    def clean_rate(self):
+        data = self.cleaned_data.get("rate", 5)
+        if data < 0 or data > 5:
```

Comment:
rate 0 thì cũng ko cho nha em
```suggestion
        if data < 1 or data > 5:
```

Reply:
Check phải là số nguyên nữa e nha